pkgs/top-level/all-packages.nix

```diff
@@ -1552,6 +1552,8 @@ in
 
   crudini = callPackage ../tools/misc/crudini { };
 
+  crytic-compile = with python3Packages; toPythonApplication crytic-compile;
```

Comment:
Does it make sense to have this in `all-packages`? Seems like it will mostly be used as a dependency of other packages, so not sure users will want to install it directly very often.

Reply:
Thanks for this suggestion.
To be honest, I have no idea if users would like to install `crytic-compile` in a standalone way.
But if desired, it is nice to have the opportunity to do so, isn't it?

Reply:
No we shouldn't pollute the top-level unnecessarily.

Reply:
@jerengie it will still be available under `python3Packages`, so it can still be installed, just not from the top-level namespace.

@FRidh the manual does say though:

> A distinction is made between applications and libraries, however, sometimes a package is used as both. In this case the package is added as a library to `python-packages.nix` and as an application to `all-packages.nix`.

Reply:
@asymmetric yes but if the author of the PR isn't going to use it like that to begin with, should we then add it just for potential users?

Reply:
I guess not. Maybe the manual should mention that the package *may* be added to `all-packages.nix`.